source/_integrations/zwave_js.markdown

```diff
@@ -1008,32 +1008,49 @@ You can switch between the official Z-Wave JS add-on and the Z-Wave JS UI add-on
 
 ### How to switch from Z-Wave JS to the Z-Wave JS UI add-on?
 
-You can switch from the official **Z-Wave JS** add-on to the **Z-Wave JS UI** add-on. However, you cannot run them both at the same time. Only one of the add-ons can be active at the same time.
+You can switch from the official **Z-Wave JS** add-on to the community **Z-Wave JS UI** add-on. However, you cannot run them both at the same time. Only one of the add-ons can be active at the same time.
 
-Switching does not require renaming your devices.
+Both add-ons communicate with Home Assistant via the same **Z-Wave** {% term integration %}.  So your devices and entities will remain unchanged if you follow these steps:
```

Comment:
```suggestion
Both add-ons communicate with Home Assistant via the same **Z-Wave** {% term integration %}.
```
I would remove this claim. Unless we are 100% percent certain nothing is changed during migration.

Reply:
I agree. We shouldn't guarantee this, since the users themselves are doing these steps. The integration is not in control of the migration.